pages/linux/goaccess.md

```diff
@@ -0,0 +1,20 @@
+# goaccess
+
+> GoAccess is an open source real-time web log analyzer.
+> More information: <https://goaccess.io>.
+
+- Run `goaccess` in interactive mode:
```

Comment:
This can probably just be:
```suggestion
- Run in interactive mode:
```

Also, for this example are there multiple files allowed? I'm not sure about the square bracket syntax for optional values.